lib/init.coffee

```diff
@@ -59,34 +60,31 @@ module.exports =
       lintOnFly: true
 
       lint: (textEditor) =>
+        rules = []
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
-        projectConfigPath = @getConfig(filePath)
+        projectConfig = @getConfig(filePath)
 
         if !fileText
           return Promise.resolve([])
 
-        parameters = [filePath]
-
-        if !projectConfigPath || !projectConfigPath.configPath
-          if !@onlyRunWhenConfig
+        if !projectConfig || !projectConfig.configPath
+          if @onlyRunWhenConfig
             atom.notifications.addError 'Pug-lint config not found'
-          return Promise.resolve([])
+            return Promise.resolve([])
```

Comment:
Shouldn't this _always_ return an empty result if no configuration was found? Currently this change will allow it to continue running when `projectConfig` is empty.